src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/documentsApiDelegateImpl/ExtractDocumentFormDataTest.java

```diff
@@ -96,4 +108,56 @@ public void withUserHavingValidRequestShouldReturnCredfated() throws IOException
         );
 
     }
+
+    @Test
+    @DisplayName("Error: non pdf throws exception")
+    public void withNonPdfFilesShouldReturnOk() throws IOException, VirusDetectedException {
+
+        Mockito.when(diligenService.postDocument(Mockito.any(), Mockito.any())).thenReturn(BigDecimal.ONE);
```

Comment:
We are duplicating this mock, I think it's better if most of them are in the `beforeall` method ? thoughts ?